src/modules/awake/Awake/Program.cs

```diff
@@ -27,26 +27,32 @@ namespace Awake
     internal class Program
     {
         private static Mutex? _mutex = null;
-        private const string AppName = "Awake";
-        private const string FullAppName = "PowerToys " + AppName;
         private static FileSystemWatcher? _watcher = null;
         private static SettingsUtils? _settingsUtils = null;
 
         public static Mutex LockMutex { get => _mutex; set => _mutex = value; }
 
         private static Logger? _log;
 
+#pragma warning disable CS8618 // Non-nullable field must contain a non-null value when exiting constructor. Consider declaring as nullable.
+        private static ConsoleEventHandler _handler;
+#pragma warning restore CS8618 // Non-nullable field must contain a non-null value when exiting constructor. Consider declaring as nullable.
```

Comment:
Why don't we follow the suggestion here? Are there reasons not to declare it nullable?

Reply:
That's because the `SetConsoleCtrlHandler` API requires a non-nullable type, therefore if I make this one nullable, I will get a compilation error.

Reply:
That's weird, it builds for me when I make it nullable